src/kms/kms.ts

```diff
@@ -919,9 +919,131 @@ export class KmsClient {
       keyWrapType,
     )
   }
+
+  /**
+   * Grant access to a KmsObject for a specific user
+   * @param uniqueIdentifier the unique identifier of the object to import
+   * @param userIdentifier the unique identifier of the user to grant access to
+   * @param operationType KMIP operation type to grant access for
+   * @returns response from KMS server
+   */
+  public async grantAccess(
```

Comment:
LGTM. 
Just a comment: 
- `grandAccess` and `revokeAccess` are very similar
- same for `listAccess`, `listOwnedObjects` and `listObtainedObjects`

Also, should we return an object where the JSON response is parsed? (I guess not but just wondering)

Reply:
I think it's more intuitive to keep seperated functions - and not passe the action as a parameter - but we can do differently if you want.
Same for the result - as you wish :)